### PR TITLE
Thang/remove mlflow from train script

### PR DESCRIPTION
**Pull request by HDThang**
- Remove mlflow code in run_plm script due to the error when run the script by model runner.
- The error maybe caused by re-define the tracking uri in the script. To test my theory, I move the mlflow code to run_clm and add the log parameters.

### CODE REVIEW

File: examples/pytorch/language-modeling/run_clm.py [hunk 1]
```diff
@@ -571,6 +585,14 @@ def compute_metrics(eval_preds):
         else None,
     )
 
+    # Mlflow initial
+    experiment_name =f'language-modeling-plm-{model_args.model_name_or_path}'
```
Comment (dohyung-kim-moreh) [inline examples/pytorch/language-modeling/run_clm.py]: `ModelRunner` set this experiment_name instead. 
You can check https://github.com/moreh-dev/moreh-testbench/blob/36adaee8bf946a62b524f4f6bc6248b175e4045a/testbench/model_runner.py#L266C18-L266C18

So, you don't need to set the experiment_name in training scripts.

Comment (HDThang) [reply examples/pytorch/language-modeling/run_clm.py]: Thanks, I've remove the experiment name that line.

**Comment by dohyung-kim-moreh** [comment]
LGTM

**Review by kimmorehvn — APPROVED** [review]
LGTM

File: examples/pytorch/language-modeling/run_clm.py [hunk 2]
```diff
@@ -619,7 +639,7 @@ def compute_metrics(eval_preds):
             kwargs["dataset"] = f"{data_args.dataset_name} {data_args.dataset_config_name}"
         else:
             kwargs["dataset"] = data_args.dataset_name
-
+    mlflow.end_run()
```
Comment (yeomni) [inline examples/pytorch/language-modeling/run_clm.py:642]: I couldn't find where mlflow.start_run() is within this script. Is it located somewhere else? Or it just needs to be deleted

Comment (HDThang) [reply examples/pytorch/language-modeling/run_clm.py:642]: The mlflow.start_run() is located in the model_runner.

**Pull request merged**